engines/sci/engine/kgraphics32.cpp

```diff
@@ -360,29 +360,112 @@ reg_t kScrollWindow(EngineState *s, int argc, reg_t *argv) {
 }
 
 reg_t kScrollWindowCreate(EngineState *s, int argc, reg_t *argv) {
-	debug("kScrollWindowCreate");
-	kStub(s, argc, argv);
-	return argv[0];
+	reg_t object = argv[0];
```

Comment:
Not something I generally do, because there are too many cases where I'd really want to say "constant after these few steps" rather than "constant immediately after first assignment". But I can see the point if you want to use it where possible.